src/commands.py

```diff
@@ -13,6 +13,9 @@
     "Remove a custom message: `{prefix}remove NAME`\n"
     "Speak a message as the bot: `{prefix}say CHAN_ID message`\n"
     "View your XP: `{prefix}xp`\n"
+    "\nAdd a game to be announced: `{prefix}addgame game_info`\n"
```

Comment:
I was keeping these in alphabetical order, but I'm open to having them separated by functionality

Reply:
Oh, didn't notice that. Don't mind either way - let me know how you want them to be ordered.

Reply:
I think it's fine as it is now, but I reserve the right to quietly reorder them.

Reply:
👍